src/main/java/frc/robot/maps/subsystems/HonreMap.java

```diff
@@ -0,0 +1,106 @@
+        final CSSparkMax rearLeftSteer = new CSSparkMax(4, MotorType.kBrushless);
+        final CSSparkMax rearRightSteer = new CSSparkMax(2, MotorType.kBrushless);
+
+        frontLeftSteer.getMotorController().setPeriodicFramePeriod(PeriodicFrame.kStatus3, 1000);
+        frontRightSteer.getMotorController().setPeriodicFramePeriod(PeriodicFrame.kStatus3, 1000);
+        rearLeftSteer.getMotorController().setPeriodicFramePeriod(PeriodicFrame.kStatus3, 1000);
+        frontRightSteer.getMotorController().setPeriodicFramePeriod(PeriodicFrame.kStatus3, 1000);
+
+        // All Distances are in Meters
+        // Front Left Module
+        final CANCoder encoderFL = new CANCoder(4);
+        encoderFL.configMagnetOffset(-36.0078125);
+        encoderFL.configAbsoluteSensorRange(AbsoluteSensorRange.Unsigned_0_to_360);
+        final SDSSwerveModule frontLeft = new SDSSwerveModule(new Translation2d(MODULE_OFFSET_XY, MODULE_OFFSET_XY),
+                encoderFL, frontLeftSteer, new CSSparkMax(7, MotorType.kBrushless),
+                SDSSwerveModule.MK3_FAST);
+
+        // Front Right Module
+        final CANCoder encoderFR = new CANCoder(3);
+        encoderFR.configMagnetOffset(-293.02734375000006);
+        encoderFR.configAbsoluteSensorRange(AbsoluteSensorRange.Unsigned_0_to_360);
+        final SDSSwerveModule frontRight = new SDSSwerveModule(new Translation2d(MODULE_OFFSET_XY, -MODULE_OFFSET_XY),
+                encoderFR, frontRightSteer, new CSSparkMax(5,
+                        MotorType.kBrushless),
+                SDSSwerveModule.MK3_FAST);
+
+        // Rear Left Module
+        final CANCoder encoderRL = new CANCoder(2);
+        encoderRL.configMagnetOffset(-102.6562);
+        encoderRL.configAbsoluteSensorRange(AbsoluteSensorRange.Unsigned_0_to_360);
+        final SDSSwerveModule rearLeft = new SDSSwerveModule(new Translation2d(-MODULE_OFFSET_XY, MODULE_OFFSET_XY),
+                encoderRL, rearLeftSteer, new CSSparkMax(3,
+                        MotorType.kBrushless),
+                SDSSwerveModule.MK3_FAST);
+
+        // Rear Right Module
+        final CANCoder encoderRR = new CANCoder(1);
+        encoderRR.configMagnetOffset(-269.121);
+        encoderRR.configAbsoluteSensorRange(AbsoluteSensorRange.Unsigned_0_to_360);
+        final SDSSwerveModule rearRight = new SDSSwerveModule(new Translation2d(-MODULE_OFFSET_XY, -MODULE_OFFSET_XY),
+                encoderRR, rearRightSteer, new CSSparkMax(1,
+                        MotorType.kBrushless),
+                SDSSwerveModule.MK3_FAST);
+
+        final double maxDriveSpeedMetersPerSecond = Units.feetToMeters(10);
+
+        final double maxRotationRadianPerSecond = Math.PI;
+
+        final DrivePID pid = new DrivePID(0.2, 0, 0.05, 0.001, 0, 0);
+
+        final Transform3d cameraPosition = new Transform3d(
+                new Translation3d(0, 0, 0),
+                new Rotation3d());
```

Comment:
This can get updated with the info Connor posted in Discord. (Note that this is in meters and he gave the distance in inches).